apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -59,12 +59,13 @@ async function canReadTickets (args) {
 }
 
 async function canManageTickets (args) {
-    const { authentication: { item: user }, operation, itemId, originalInput, context, listKey } = args
-
+    const { authentication: { item: user }, operation, itemId, itemIds, originalInput, context, listKey } = args
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin) return true
 
+    const isBulkRequest = Array.isArray(originalInput)
```

Comment:
It would be cool if we returned `false` below if it is bulk request and user is not `staff` and it is not an `update` operation (since we handle only this case). After `directAccess` check

Reply:
Good idea, I'll do that